variant_calling_benchmarks/joint_caller/process_results.py

```diff
@@ -128,7 +135,7 @@ def parse_mixture_likelihoods(strings):
 
 def expand_sample_info_columns_one_row(full_row, result):
     sample_info = full_row['sample_info']
-    for (sample, info) in sample_info.iteritems():
+    for (sample, info) in six.iteritems(sample_info):
```

Comment:
Cool, thanks for the `py3` fixes. Using just `.items()` here would also be fine I think, either way works for me